d2bs/kolbot/libs/bots/AncientTunnels.js

```diff
@@ -9,7 +9,9 @@ function AncientTunnels() {
 	Pather.useWaypoint(44);
 	Precast.doPrecast(true);
 
-	if (Config.AncientTunnels.OpenChest && Pather.moveToPreset(me.area, 2, 580) && Misc.openChests(5)) {
+	if (Config.AncientTunnels.OpenChest && Pather.moveToPreset(me.area, 2, 580)) {
+		Chest.scan(5);
+		Chest.openChests();
```

Comment:
I don't see the motivation behind separating scan from openChests(), also this change the logic where previously it would not pickItems unless openChests returns true (presumably no chest found otherwise)

Reply:
* re-added the 3rd condition on line 12
* removed lines 13, 14
* will modify Chest.openChests to take a parameter for scan distance